docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+services:
+  app:
+    container_name: berth-reservation-admin
+    build: .
+    volumes:
+      - .:/home/node/app
+      - /home/node/app/node_modules
+    working_dir: /home/node/app
```

Comment:
Is there a need for this? I have not needed this myself for any project so far. This should be specified by the WORKDIR in the Dockerfile IMHO.

Reply:
Seems redundant and has been removed.